### PR TITLE
zh: improve deploy-on-gcp-gke.md as in aws doc

### PR DESCRIPTION
**Pull request by dragonly**
### What is changed, added or deleted? (Required)

improve GCP GKE doc

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

None

### Do your changes match any of the following descriptions?

None

### CODE REVIEW

File: zh/deploy-on-gcp-gke.md [hunk 1]
```diff
@@ -8,24 +8,22 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-gcp-gke/']
...
 * [gcloud](https://cloud.google.com/sdk/gcloud)：用于创建和管理 GCP 服务的命令行工具
 * 完成 [GKE 快速入门](https://cloud.google.com/kubernetes-engine/docs/quickstart#before-you-begin) 中的**准备工作** (Before you begin)
 
-    该教程包含以下内容：
+该教程包含以下内容：
 
-    * 启用 Kubernetes API
-    * 配置足够的配额等
+* 启用 Kubernetes API
+* 配置足够的配额等
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md:24]: As in the AWS doc, we shall indent these lines.

File: zh/deploy-on-gcp-gke.md [hunk 2]
```diff
@@ -36,98 +34,112 @@ gcloud config set compute/region <gcp-region>
 
 使用以上命令，设置好你的 GCP 项目和默认的区域。
 
-### 创建 GKE 集群
+## 创建 GKE 集群和节点池
 
-1. 创建 GKE 集群和一个默认节点池：
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md:41]: The ordered list is necessary for this section.

It helps users quickly pick out **steps** from intro/ref, etc.

File: zh/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -8,24 +8,22 @@ aliases: ['/docs-cn/tidb-in-kubernetes/dev/deploy-on-gcp-gke/']
 
 <!-- markdownlint-disable MD029 -->
 
-本文介绍了如何在 GCP GKE 上部署 TiDB 集群。
+本文介绍了如何部署 GCP GKE 集群，并在其中部署 TiDB 集群。
```
Comment (ran-huang) [inline zh/deploy-on-gcp-gke.md:11]:
```suggestion
本文介绍了如何部署 GCP Google Kubernetes Engine(GKE) 集群，并在其中部署 TiDB 集群。
```
For less commonly used acronyms, let's spell them out at the first appearance.

File: zh/deploy-on-gcp-gke.md (hunk 2 above)
Comment (DanielZhangQD) [reply zh/deploy-on-gcp-gke.md:41]: Agree

File: zh/deploy-on-gcp-gke.md [hunk 4]
```diff
@@ -36,98 +34,112 @@ gcloud config set compute/region <gcp-region>
...
+下面介绍如何在 GCP GKE 上部署 TiDB 集群和监控组件。
 
-    ```shell
-    curl -LO https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-cluster.yaml &&
-    curl -LO https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/gcp/tidb-monitor.yaml
-    ```
+### 部署 CRDs
 
-2. 创建 `Namespace`：
+执行以下命令，在 GKE 集群中部署 TidbCluster 和 TidbMonitor CR：
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md:76]: 只是 download 了 yaml

File: zh/deploy-on-gcp-gke.md [hunk 5]
```diff
@@ -136,75 +148,75 @@ gcloud compute instances create bastion \
...
 >
-> `<your-region>-a` 为集群所在的区域的 a 可用区，比如 us-central1-a。也可在同区域下的其他可用区创建堡垒机。
+> `${your-region}-a` 为集群所在的区域的 a 可用区，比如 us-central1-a。也可在同区域下的其他可用区创建堡垒机。
 
 ### 安装 MySQL 客户端并连接
 
 待创建好堡垒机后，我们可以通过 SSH 远程连接到堡垒机，再通过 MySQL 客户端来访问 TiDB 集群。
 
-1. 用 SSH 连接到堡垒机：
+用 SSH 连接到堡垒机：
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md:162]: Suggest keeping the number list here.

**Comment by dragonly** [comment]
suggestions applied

**Review by ran-huang — COMMENTED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

File: zh/deploy-on-gcp-gke.md [hunk 6]
```diff
@@ -56,92 +54,106 @@ gcloud config set compute/region <gcp-region>
...
+此过程可能需要几分钟。
 
-参考快速上手中[部署 TiDB Operator](get-started.md#部署-tidb-operator)，将 TiDB Operator 部署进 Kubernetes 集群。
+## 部署 TiDB Operator
 
-### 部署 TiDB 集群和监控
+参考快速上手中[部署 TiDB Operator](get-started.md#部署-tidb-operator)，在 GKE 集群中部署 TiDB Operator。
 
-1. 准备 TidbCluster 和 TidbMonitor CR 文件：
+## 部署 TiDB 集群和监控
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md:70]: 参考 EKS 文档改动 comments。

Comment (dragonly) [reply zh/deploy-on-gcp-gke.md:70]: fixed

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**